graphix/pattern.py

```diff
@@ -641,13 +641,13 @@ def connected_edges(self, node, edges):
         return connected
 
     def _measurement_order_space(self):
-        """Determine the optimal measurement order
-         which minimize the `max_space` of the pattern.
+        """Determine a sub-optimal measurement order
```

Comment:
entire doctoring should read:

Determine measurement order that heuristically optimises the `max_space` of a pattern